views/overlays/springshowdown/game4by3.hbs

```diff
@@ -89,6 +104,28 @@
         }
       }
 
+      $('#lplayer').bind('newName', function(e, newText){
+        $(this).animate({'opacity': '0', 'margin-left': '-1%'}, 350, function() {
+            $(this).text(newText);
+          }).animate({'opacity': '1', 'margin-left': '0'}, 350);
+
+         scaleFont(document.getElementById('lplayer'), newText.length);
+      });
```

Comment:
Yup. I'll also probably refactor to use the jQuery dom object instead of vanilla's, and that will make this look a lot cleaner.